internal/account/idgenerator_test.go

```diff
@@ -34,6 +34,8 @@ func Test_GenerateRandomID(t *testing.T) {
 		g.Expect(id1).ToNot(gomega.Equal(id2))
 	})
 
+	// if you use apple silicon, run this `go env -w GOARCH=amd64`
```

Comment:
Thanks bro, I don't know monkey would make a problem for mac users 🙏

Reply:
no problem mate ❤️